MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -506,6 +507,12 @@ private void listItems(int id) {
           R.string.activity_mas_directions_url
         ));
         exampleItemModel.add(new ExampleItemModel(
+          R.string.activity_mas_optimization_title,
+          R.string.activity_mas_optimization_description,
+          new Intent(MainActivity.this, OptimizationActivity.class),
+          R.string.activity_mas_directions_url
```

Comment:
Please change to `activity_mas_optimization_url` for continuity with `activity_mas_optimization_title` and `activity_mas_optimization_description` strings